installers/linux-deb/build.sh

```
@@ -53,46 +53,109 @@ fi
+# Move distributions contents to target (without JDK)
+mkdir -p "$BALLERINA_TARGET"
+mv "$BALLERINA_TEMP"/* "$BALLERINA_TARGET"
+
+# Move JDK to shared dependencies directory
+print_info "Moving JDK to shared dependencies directory"
+rm -rf "$DEPENDENCIES_DIR"
+mkdir -p "$DEPENDENCIES_DIR"
+if [ -d "$BALLERINA_UNZIPPED_PATH/dependencies" ]; then
+    for jdk_folder in "$BALLERINA_UNZIPPED_PATH/dependencies"/*; do
+        if [ -d "$jdk_folder" ]; then
+            JDK_FOLDER=$(basename "$jdk_folder")
```

Shellcheck (0.11.0)

[warning] JDK_FOLDER appears unused. Verify use (or export if used externally).

(SC2034)